### ISSUE TITLE
import cupy 9.0.0a2 doesn't work on Windows 10

### ISSUE THREAD

**Issue by ma-sadeghi** [cupy/cupy#4352]
Thank you for using and contributing to CuPy!

If your issue is a **request for support in using CuPy**, please post it on [Stack Overflow](https://stackoverflow.com/questions/tagged/cupy) or Google Groups ([en](https://groups.google.com/forum/#!forum/cupy) / [ja](https://groups.google.com/forum/#!forum/cupy-ja)).
Developer team members are monitoring questions on these channels.

If it is a bug report, please include the following information:

* Conditions (you can just paste the output of `python -c 'import cupy; cupy.show_config()'`)
  - CuPy version **9.0.0a2**
  - OS/Platform: **Windows 10**
  - CUDA version: **10.2**
  - cuDNN/NCCL version (if applicable)
* Code to reproduce
```python
>>> import cupy
```
* Error messages, stack traces, or logs
```bash
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\Amin\anaconda3\envs\py38\lib\site-packages\cupy\__init__.py", line 57, in <module>
    from cupy import fft  # NOQA
  File "C:\Users\Amin\anaconda3\envs\py38\lib\site-packages\cupy\fft\__init__.py", line 1, in <module>
    from cupy.fft._fft import fft  # NOQA
  File "C:\Users\Amin\anaconda3\envs\py38\lib\site-packages\cupy\fft\_fft.py", line 9, in <module>
    from cupy.fft import config
  File "C:\Users\Amin\anaconda3\envs\py38\lib\site-packages\cupy\fft\config.py", line 13, in <module>
    from cupy.fft._callback import get_current_callback_manager  # NOQA
  File "cupy\fft\_callback.pyx", line 35, in init cupy.fft._callback
AttributeError: 'NoneType' object has no attribute 'split'
```

**Comment by leofang** [comment]
Hi @ma-sadeghi Thanks for reporting! This is surprising, could you help me debug it? I need to know the output of the following snippet on Windows:
```python
import sysconfig
sysconfig.get_config_var('CXX')
```

**Comment by ma-sadeghi** [comment]
No worries, it doesn't return anything for me.

**Comment by leofang** [comment]
Thanks! Right, this callback module is new, so it's not included in the v8 releases. I will push a fix.

**Comment by leofang** [comment]
btw forgot to say, though this is irrelevant to my planned fix (as the callback doesn't work on Windows anyway): Any chance you know the path to an MSVC compiler installed on your system? I couldn't make sense of the output you reported.

**Comment by ma-sadeghi** [comment]
I don't think I have MSVC compiler installed

**Comment by kmaehashi** [comment]
Thank you for the feedback, we'll soon make a release including this fix as v9.0.0a3.

**Comment by leofang** [comment]
> I don't think I have MSVC compiler installed

Hi @ma-sadeghi I know nothing about CUDA + Windows, so I took a quick look at the [Installation Guide](https://docs.nvidia.com/cuda/cuda-installation-guide-microsoft-windows/index.html) and found that MSVC needs to be installed before installing CUDA Toolkit. Looks like you're using a conda environment, but given that we haven't built Windows packages yet (https://github.com/conda-forge/cupy-feedstock/issues/62), I assume that you already have a CUDA Toolkit installed locally, and then install CuPy by wheel (`pip install cupy-cuda102`)? But this means you should have MSVC somewhere, or NVCC would not work. I am wondering how to make Python discover where your MSVC is...

**Comment by ma-sadeghi** [comment]
@leofang You're right, I have Cuda Toolkit installed on conda (from https://anaconda.org/anaconda/cudatoolkit), and then I installed cupy using `pip` in my conda environment. 

When I run `nvcc --version` in `cmd` or conda prompt, I get a valid output:

```bash
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2020 NVIDIA Corporation
Built on Mon_Oct_12_20:54:10_Pacific_Daylight_Time_2020
Cuda compilation tools, release 11.1, V11.1.105
Build cuda_11.1.relgpu_drvr455TC455_06.29190527_0
```

I'm not really familiar with C++ build tools, but I just installed the C++ build tools from Microsoft, and still I get no output when running:

```python
import sysconfig
sysconfig.get_config_var('CXX')
```

**Comment by kmaehashi** [comment]
> we'll soon make a release including this fix as v9.0.0a3.

Unfortunately, as many test failures are observed in #4362, we decided not to release v9.0.0a3 this time and will work on fixing the Windows issues towards v9.0.0b1 planned next month. We are going to take down v9.0.0a2 Windows wheels. We're sorry for the inconvenience.

**Comment by kmaehashi** [comment]
This is everything I could obtain via `sysconfig` module in my test environment:
```
C:\Users\chainer\cupy>python -m sysconfig
Platform: "win-amd64"
Python version: "3.7"
Current installation scheme: "nt"

Paths:
        data = "C:\Development\Python\Python37"
        include = "C:\Development\Python\Python37\Include"
        platinclude = "C:\Development\Python\Python37\Include"
        platlib = "C:\Development\Python\Python37\Lib\site-packages"
        platstdlib = "C:\Development\Python\Python37\Lib"
        purelib = "C:\Development\Python\Python37\Lib\site-packages"
        scripts = "C:\Development\Python\Python37\Scripts"
        stdlib = "C:\Development\Python\Python37\Lib"

Variables:
        BINDIR = "C:\Development\Python\Python37"
        BINLIBDEST = "C:\Development\Python\Python37\Lib"
        EXE = ".exe"
        EXT_SUFFIX = ".pyd"
        INCLUDEPY = "C:\Development\Python\Python37\Include"
        LIBDEST = "C:\Development\Python\Python37\Lib"
        SO = ".pyd"
        VERSION = "37"
        abiflags = ""
        base = "C:\Development\Python\Python37"
        exec_prefix = "C:\Development\Python\Python37"
        installed_base = "C:\Development\Python\Python37"
        installed_platbase = "C:\Development\Python\Python37"
        platbase = "C:\Development\Python\Python37"
        prefix = "C:\Development\Python\Python37"
        projectbase = "C:\Development\Python\Python37"
        py_version = "3.7.2"
        py_version_nodot = "37"
        py_version_short = "3.7"
        srcdir = "C:\Development\Python\Python37"
        userbase = "C:\Users\chainer\AppData\Roaming\Python"
```
